src/chigame/users/urls.py

```diff
@@ -2,9 +2,12 @@
 
 from chigame.users.views import user_detail_view, user_redirect_view, user_update_view
 
+from . import views
+
 app_name = "users"
 urlpatterns = [
     path("~redirect/", view=user_redirect_view, name="redirect"),
     path("~update/", view=user_update_view, name="update"),
     path("<int:pk>/", view=user_detail_view, name="detail"),
+    path("users/user_detail.html", views.user_list, name="user_list"),
```

Comment:
This path should still be "user-list" like before because you want your final url to look like http://127.0.0.1:8000/users/user-list/. I was referencing line 51 in views.py. The path to the template is wrong there –– it should be "users/user_detail.html".

Reply:
I see, thank you for clarifying for me. I just pushed my changes, tested the server, and it works on my side now. Thank you for your corrections.